pkg/infra/usagestats/usage_stats.go

```diff
@@ -99,6 +99,28 @@ func (uss *UsageStatsService) GetUsageReport(ctx context.Context) (UsageReport,
 		} else {
 			dsOtherCount += dsStat.Count
 		}
+
+		if dsStat.Type == models.DS_ES {
+			dsSettings := models.GetESJSONDataQuery{}
+			if err := uss.Bus.Dispatch(&dsSettings); err != nil {
+				metricsLogger.Error("Failed to get elasticsearch json data", "error", err)
+				return report, err
+			}
+
+			for _, data := range dsSettings.Result {
+				var m models.ESJSONData
+
+				if err := json.Unmarshal([]byte(data.JsonData), &m); err != nil {
+					return report, err
+				}
+
+				statName := fmt.Sprintf("stats.ds.elasticsearch.v%d.count", m.ESVersion)
+
+				count, _ := metrics[statName].(int64)
```

Comment:
if `metrics[statName]` is not an `int64`, this will make `count=0`. is that what we want?

Reply:
Yep, the idea is to increment the counter starting from 0, since we are creating "stats.ds.elasticsearch.v%d.count"
only here, on the first iteration of each es version that count would start at 0